xtask/src/cli.rs

```diff
@@ -117,26 +117,29 @@ impl Subcommand {
 }
 
 pub fn run(command: &mut Command) -> Result<()> {
-    println!("Running: {}", format_command(command));
+    let command_str = format_command(command);
+    let github_actions = std::env::var_os("GITHUB_ACTIONS").is_some();
```

Comment:
It might be better to do this as part of parsing `Command`, especially if other parts of the program also need to know in which environment they're running.

Reply:
Oh I see you already merged 😅 It's not a big deal, but we might want to change it at some point.